extensions/msal-node-extensions/src/broker/NativeBrokerPlugin.ts

```diff
@@ -487,6 +489,23 @@ export class NativeBrokerPlugin implements INativeBrokerPlugin {
                     }
                 );
             }
+
+            const skus =
+                request.extraParameters &&
+                request.extraParameters[AADServerParamKeys.X_CLIENT_EXTRA_SKU]
+                    ?.length
+                    ? request.extraParameters[
+                          AADServerParamKeys.X_CLIENT_EXTRA_SKU
+                      ]
+                    : "";
+            authParams.SetAdditionalParameter(
+                AADServerParamKeys.X_CLIENT_EXTRA_SKU,
+                ServerTelemetryManager.makeExtraSkuString({
+                    skus,
+                    extensionName: "msal.node.ext",
```

Comment:
Can we make this a constant?

Reply:
Unfortunately, we can't as `skus` is picked up from `request.extraParameters`.

Reply:
I was referring to `msal.node.ext`